plugins/renamerOnUpdate/renamerOnUpdate.py

```diff
@@ -461,6 +461,8 @@ def makeFilename(scene_information, query):
     sys.exit(1)
 
 # Looking for duplicate filename
+new_filename = new_filename.replace("'", "''")  # Fixing a bug whereby filenames with apostrophes would end the string prematurely, and would throw a syntax error
+                                                #when executing the query
 sqlite_query = "SELECT id FROM scenes WHERE path LIKE '%{}%';".format(new_filename)
 cursor.execute(sqlite_query)
```

Comment:
probably the [safer method](https://docs.python.org/3/library/sqlite3.html#sqlite3-placeholders)
```suggestion
sqlite_query = "SELECT id FROM scenes WHERE path LIKE ?;"
sqlite_args = ("%{}%".format(new_filename), )
cursor.execute(sqlite_query, sqlite_args)
```